code/modules/research/xenobiology/crossbreeding/_clothing.dm

```diff
@@ -44,6 +44,7 @@ Slimecrossing Armor
 /obj/item/clothing/glasses/prism_glasses/Initialize(mapload)
 	. = ..()
 	AddElement(/datum/element/wearable_client_colour, /datum/client_colour/glass_colour, ITEM_SLOT_EYES, glasses_color, forced_glass_color)
+	AddComponent(/datum/component/item_equipped_movement_rustle, SFX_PLATE_ARMOR_RUSTLE, 8)
```

Comment:
♂ Hey buddy, I think you've got the wrong item, the adamantine armor is 100 lines down. ♂

Reply:
Oops